Cargo.toml

```diff
@@ -107,10 +108,13 @@ members = ["crates/*", "sdk/rust", "sdk/rust/macro"]
 
 [workspace.dependencies]
 tracing = { version = "0.1", features = ["log"] }
-wasmtime-wasi = { version = "13.0.0", features = ["tokio"] }
-wasi-common-preview1 = { version = "13.0.0", package = "wasi-common" }
-wasmtime = { version = "13.0.0", features = ["component-model"] }
-spin-componentize = { git = "https://github.com/fermyon/spin-componentize", rev = "f29cdb26a6b1700ae1fe48c94bbef26a8069d566" }
+wasmtime-wasi-http = { git = "https://github.com/bytecodealliance/wasmtime", rev = "17505667fa2895b3bd31bb9f5f4c544250090926" }
+wasmtime-wasi = { git = "https://github.com/bytecodealliance/wasmtime", rev = "17505667fa2895b3bd31bb9f5f4c544250090926", features = ["tokio"] }
+wasi-common-preview1 = { git = "https://github.com/bytecodealliance/wasmtime", rev = "17505667fa2895b3bd31bb9f5f4c544250090926", package = "wasi-common" }
+wasmtime = { git = "https://github.com/bytecodealliance/wasmtime", rev = "17505667fa2895b3bd31bb9f5f4c544250090926", features = ["component-model"] }
+spin-componentize = { git = "https://github.com/fermyon/spin-componentize", branch = "wasmtime-17505667" }
```

Comment:
Let's make sure to get the changes we need on `main` in spin-componentize.

Reply:
That Wasmtime commit _is_ from the `release-14.0.0` branch (as it existed when I did the update).  I considered just pointing to the `release-14.0.0` branch, but I was a bit paranoid about various Cargo.lock files getting out of sync.

Reply:
Also, the Wasmtime version really has to match the adapter _exactly_, or you get build errors, so I didn't want to imply that any snapshot of `release-14.0.0` would necessarily work.